ci-operator/config/openshift/loki/openshift-loki-release-5.8.yaml

```
@@ -0,0 +1,153 @@
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+  opa-openshift:
+    name: mirror-5.y
+    namespace: logging
+    tag: opa-openshift
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.14
+  to: loki
+- dockerfile_path: Dockerfile.promtail.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.14
+  to: promtail
+- context_dir: operator
+  dockerfile_path: Dockerfile
+  from: base
+  to: loki-operator
+operator:
+  bundles:
+  - as: loki-operator-bundle
+    context_dir: operator/bundle/openshift
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift-logging/loki-operator:0.1.0
+    with: pipeline:loki-operator
+  - pullspec: quay.io/openshift-logging/loki:v2.9.1
+    with: pipeline:loki
+  - pullspec: quay.io/observatorium/api:latest
+    with: pipeline:lokistack-gateway
+  - pullspec: quay.io/observatorium/opa-openshift:latest
+    with: pipeline:opa-openshift
+promotion:
+  additional_images:
+    loki-operator-bundle: loki-operator-bundle
+  name: "5.8"
+  namespace: logging
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: lint
+  steps:
+    test:
+    - as: lint
+      commands: cd operator && GOFLAGS="" make lint
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: test
+  steps:
+    test:
+    - as: unit
+      commands: cd operator && GOFLAGS="" go test ./...
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - as: prometheus-rules
+      commands: cd operator && make test-unit-prometheus
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: build
+  steps:
+    test:
+    - as: manager
+      commands: cd operator && GOFLAGS="" make manager
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - as: broker
+      commands: cd operator && GOFLAGS="" make bin/loki-broker
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - as: storage-size-calculator
+      commands: cd operator && GOFLAGS="" make size-calculator
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: test-operator
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
```

Should we try to test on 4.13 (and 4.14 after their release)?

```suggestion
    version: "4.13"
```